tests/csi-sanity/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM golang
 #ENV test=test
-RUN go get github.com/kubernetes-csi/csi-test/cmd/csi-sanity
+RUN GO_ENABLED=0 GOOS=linux GOARCH=amd64 go get -u -t github.com/kubernetes-csi/csi-test/cmd/csi-sanity || true
```

Comment:
@tigrawap, I've recalled the problem I have had with csi-sanity test in container.
Apparently, the `go get github.com/kubernetes-csi/csi-test/cmd/csi-sanity` command constantly fails due to error that the package contains only tests

I could only find some info on this here: https://github.com/golang/go/issues/29268#
I've tried to somehow work around it - but it fails (not only returns invalid retcode, but actually doesn't install the package)
Ideas are welcome!